devlib/native/win_native.cpp

```diff
@@ -168,6 +169,53 @@ namespace winutil {
     }
 
 
+    static auto findBusNumber(DEVINST const& handle, QMap<QString, int> & cachedDeviceBuses) {
+        static auto rootHubsBuses = initializeRootBuses();
+        qDebug() << "Buses" << rootHubsBuses;
```

Comment:
You have the `QLoggingCategory` defined, so I'd rather use `qCDebug(winlog())`

Reply:
Ok